### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR, Veteran New Account Flow within Authenticated Experience, Welcome Pathway V1

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#83192]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** As you refine the content, be careful about link text. You'll want each link to have a unique accessible name that describes its purpose, eg. "Find out if you're eligible for [benefit]" rather than just "Find out if you're eligible."
- [x] **Must:** Echoing @artsymartha68, approaching this from a mobile-first perspective is important. That's likely to be the first-choice device for a lot of exiting servicemembers, and blind users often use mobile devices as their primary or only device thanks to robust gesture support in iOS and Android.
- [ ] **Should:** As you decide whether/how long the welcome page persists for users, make sure you disclose to users what will happen ("you'll see this page for 30 more days" or "if you ever want to view this page again, go to..." etc).
- [ ] **Consider:** If possible, consider tracking what tasks linked from the welcome page get completed by Veterans. This might trim down on the cognitive load of the page a bit so that it's not left to users to distinguish between things they've already done and things they still need to do. 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by emcruer** [comment]
1. **Must: As you refine the content, be careful about link text. You'll want each link to have a unique accessible name that describes its purpose, eg. "Find out if you're eligible for [benefit]" rather than just "Find out if you're eligible."**
- We've made sure link text is more specific to the result
2. **Must: Echoing @artsymartha68, approaching this from a mobile-first perspective is important. That's likely to be the first-choice device for a lot of exiting servicemembers, and blind users often use mobile devices as their primary or only device thanks to robust gesture support in iOS and Android.**
- We started with a mobile first prototype, then moved to desktop. We've had our CAIA consult and we are testing the concepts right now, before planning for an additional study focused on AT once we have code.
3. **Should: As you decide whether/how long the welcome page persists for users, make sure you disclose to users what will happen ("you'll see this page for 30 more days" or "if you ever want to view this page again, go to..." etc).**
- Since we have now closely collaborated with My VA and Profile, the welcome message is current a dismissible alert with suggested actions vs a page
4. **Consider: If possible, consider tracking what tasks linked from the welcome page get completed by Veterans. This might trim down on the cognitive load of the page a bit so that it's not left to users to distinguish between things they've already done and things they still need to do**.
- We also have a new component that has been presented to the Design Council which will track a simple series of tasks

Closing this ticket since the 'Musts' are complete and the 'Should/Could' have also been considered as part of our plans. Cc'ing @KKufalk13 for awareness.
